include/psa/crypto.h

```diff
@@ -985,12 +978,12 @@ static psa_hash_operation_t psa_hash_operation_init(void);
  * \retval #PSA_ERROR_INVALID_ARGUMENT
  *         \p alg is not a hash algorithm.
  * \retval #PSA_ERROR_BAD_STATE
- *         The operation state is not valid (it must be inactive).
+ *         The operation state is not valid (it must be inactive), or
+ *         the library has not been previously initialized by psa_crypto_init().
```

Comment:
The line `the library has not been previously initialized by psa_crypto_init().` was copied here (under `PSA_ERROR_BAD_STATE`), but it also still exists for `PSA_ERROR_INSUFFICIENT_MEMORY, PSA_ERROR_COMMUNICATION_FAILURE, PSA_ERROR_HARDWARE_FAILURE, PSA_ERROR_CORRUPTION_DETECTED`.
It that ok?

Reply:
That line only applies to `PSA_ERROR_BAD_STATE`. A lot of retvals have no documentation. When I wrote
```
\retval #PSA_X
\retval #PSA_Y
    Description of Y.
```
I intended PSA_X to have empty documentation, not to have the same documentation as PSA_Y.

According to the HTML rendering, the effect is what I intended. But going by the [documentation](https://www.doxygen.nl/manual/commands.html#cmdretval), I guess I was mistaken?

Reply:
Oh, I thought that the intention was in line with doxygen docs, so having multiple return codes share the same description! I'll have to rework this PR then, if that's not the case.

Reply:
Reworked it according to @gilles-peskine-arm's interpretation of return code descriptions.